app/src/main/java/fr/free/nrw/commons/Media.java

```diff
@@ -128,89 +128,179 @@ public String getImageUrl() {
+     * @param filename the new name of the file
+     */
     public void setFilename(String filename) {
         this.filename = filename;
     }
 
+    /**
+     * Gets the file description.
+     * @return file description as a string
+     */
     public String getDescription() {
         return description;
     }
 
+    /**
+     * Sets the file description.
+     * @param description the new description of the file
+     */
     public void setDescription(String description) {
         this.description = description;
     }
 
+    /**
+     * Gets the datalength of the file.
+     * @return file datalength as a long
+     */
     public long getDataLength() {
         return dataLength;
     }
 
+    /**
+     * Sets the datalength of the file.
+     * @param dataLength as a long
+     */
     public void setDataLength(long dataLength) {
         this.dataLength = dataLength;
     }
 
+    /**
+     * Gets the creation date of the file.
+     * @return creation date as a Date
+     */
     public Date getDateCreated() {
         return dateCreated;
     }
 
+    /**
+     * Sets the creation date of the file.
+     * @param date creation date as a Date
+     */
     public void setDateCreated(Date date) {
         this.dateCreated = date;
     }
 
+    /**
+     * Gets the upload date of the file.
+     * Can be null.
+     * @return upload date as a Date
+     */
     public @Nullable
     Date getDateUploaded() {
         return dateUploaded;
     }
 
+    /**
+     * Gets the name of the creator of the file.
+     * @return creator name as a String
+     */
     public String getCreator() {
         return creator;
     }
 
+    /**
+     * Sets the creator name of the file.
+     * @param creator creator name as a string
+     */
     public void setCreator(String creator) {
         this.creator = creator;
     }
 
+    /**
+     * Gets the width of the media.
+     * @return file width as an int
+     */
     public int getWidth() {
         return width;
     }
 
+    /**
+     * Sets the width of the media.
+     * @param width file width as an int
+     */
     public void setWidth(int width) {
         this.width = width;
     }
 
+    /**
+     * Gets the height of the media.
+     * @return file height as an int
+     */
     public int getHeight() {
         return height;
     }
 
+    /**
+     * Sets the height of the media.
+     * @param height file height as an int
+     */
     public void setHeight(int height) {
         this.height = height;
     }
 
+    /**
+     * Gets the license name of the file.
+     * @return license as a String
+     */
     public String getLicense() {
         return license;
     }
 
+    /**
+     * Sets the license name of the file.
+     * @param license license name as a String
+     */
     public void setLicense(String license) {
         this.license = license;
     }
 
+    /**
+     * Gets the coordinates of the file.
+     * </p>
+     * Presumably indicates upload location.
```

Comment:
It is actually the location where the picture was taken :-)

Reply:
The specified JavaDocs have been edited for accuracy.